.github/workflows/ci.yml

```diff
@@ -20,34 +20,24 @@ jobs:
           - ubuntu-18.04
           - windows-2019
           - macos-10.15
+        go:
+          - 1.14
+          - 1.15
+          - 1.18
+          - 1.17
+          - 1.18
     steps:
       -
         name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
```

Comment:
Not in this PR but we may want to update dependabot configuration to account for GHA too https://github.com/hashicorp/terraform-registry-address/blob/main/.github/dependabot.yml

Reply:
Good idea! I'll update dependabot